src/prisma/generator/templates/actions.py.jinja

```diff
@@ -131,11 +132,55 @@ class {{ model.name }}Actions:
         return {{ maybe_await }}self._client.query_first(query, *args, model=self._model)
     {% endif %}
 
+    @overload
     {{ maybe_async_def }}create(
         self,
         data: types.{{ model.name }}CreateInput,
-        include: Optional[types.{{ model.name}}Include] = None
+        include: Optional[types.{{ model.name}}Include],
+        discard_result: Literal[False] = False,
+    ) -> {{ ModelType }}:
+        ...
+
+    @overload
+    {{ maybe_async_def }}create(
+        self,
+        data: types.{{ model.name }}CreateInput,
+        include: Optional[types.{{ model.name}}Include],
+        discard_result: Literal[True],
+    ) -> None:
+        ...
+
+    @overload
+    {{ maybe_async_def }}create(
+        self,
+        data: types.{{ model.name }}CreateInput,
+        *,
+        discard_result: Literal[True],
+    ) -> None:
+        ...
+
+    @overload
+    {{ maybe_async_def }}create(
+        self,
+        data: types.{{ model.name }}CreateInput,
+        *,
+        discard_result: Literal[False],
```

Comment:
Looks like you ran into some issues with the ordering of arguments with defaults. If you mark all arguments after `data` as keyword only you should be able to remove some overloads and keep the `None` default for `include`.

Reply:
@RobertCraigie Sorry I'm not sure I follow.  Are you suggesting enforcing keywords by changing the `create` signature to:
```python
async def create(
        self,
        data: types.PostCreateInput,
        *,
        include: Optional[types.PostInclude] = None,
        discard_result: bool = False,
    ) -> Optional['models.Post']:
```

Or is the suggestion strictly about the overloads themselves?